src/patternfly/demos/Tabs/examples/Tabs.md

```diff
@@ -0,0 +1,814 @@
+---
+id: 'Tabs'
+section: components
+---
+
+## Examples
+
+### Open tabs
+
+```hbs isFullscreen
+{{#> tabs--page-wrapper tabs--page-wrapper--id="open-tabs-example"}}
+  {{> page-template-breadcrumb page-template-breadcrumbs--IsPods="true"}}
+  {{#> page-main-section page-main-section--IsLimitWidth="true" page-main-section--modifier="pf-m-light"}}
+    {{> tabs--pod-page-title}}
+  {{/page-main-section}}
+  {{#> page-main-tabs page-main-tabs--IsLimitWidth="true" page-main-tabs--modifier="pf-m-light pf-m-tabs"}}
```

Comment:
Do you need that? Looks like you set `background-color` on `.pf-c-page__main-tabs`. If not, it's in other spots in the demos, too.

```suggestion
  {{#> page-main-tabs page-main-tabs--IsLimitWidth="true" page-main-tabs--modifier="pf-m-tabs"}}
```